esp/scm/ws_topology.ecm

```diff
@@ -316,6 +316,15 @@ ESPStruct TpClusterNameType
     bool   IsDefault;
 };
 
+ESPStruct TpStoragePlan
+{
+    string Name;
+    string Hosts;
+    string Prefix;
+    string Mount;
+    unsigned NumDevices;
```

Comment:
I am not sure this is the information we should be pulling out of the configuration and exposing via a service.
(it may be)

We'll want to present a list of LandingZones I think, which will probably be a select list of storage planes,
but not sure if we want to expose all the planes and/or all their properties.

We might, I'm not sure, but I suspect there comes a point where you just want to view the configuration as a whole, without a strict EspStruct that is incomplete, or can go out of sync.
FWIW the source reference for a storage plane is : https://github.com/hpcc-systems/HPCC-Platform/blob/b1167eeee37e60ac72e51d20e11918ebb3294b78/helm/hpcc/values.schema.json#L303

This is also probably true of other config information.
Do we want a discreet service for each part that duplicates the schema's in esp struct responses, or customized them somehow, or do we only want very select parts for specialized reasons (like a list of LZ's)...
And then for detailed configuration details - embed or view them via a K8s type dashboard.

Reply:
@ghalliday - can you comment?

Reply:
My question is what is the purpose of this service - I looked in the jira and couldn't see any reasons/requirements.

drop zones lists and targets for sprays should/will be provided by the standard calls.  Who is expected to calls this, and why?

Reply:
The purpose of this service method is to give a user some general information about storage planes. Operation team may check the information from ECLWatch or SOAP calls.

Reply:
Does the user need to know about storage planes as a logical concept?  They need to know about dropzones (already returned from one function), and locations that data can be copied to/stored.  I would have thought there was a function to provide that as well already.

Reply:
Do we expect some users know about storage planes as a logical concept? Do we already mention the storage planes in our documents (like exportingData.rst)?
The existing TpDropZoneQuery only works for bare metal environment. I may upgrade it to support cloud environment. To do that, I need your advice about how to identify the storage planes used as dropzones from other storage planes.
If we can make the TpDropZoneQuery works for cloud environment, please let me know if you think we do not need this PR any more so that I can close it.

Reply:
>To do that, I need your advice about how to identify the storage planes used as dropzones from other storage planes.

@shamser is working on a PR, that will tag storage planes as LZ's.
There will then be a helper function that will return those LZ's I think.

Reply:
@jakesmith @wangkx I have added an attribute under plane called isDropZone.  By the way, I have made changes to WsTopology::getTpDropZones.  I'll expect to submit a PR today with the changes.

Reply:
Thanks shamser.

Reply:
@wangkx Here's my changes relating to dfuserver & dropzones: https://github.com/hpcc-systems/HPCC-Platform/pull/14605